crates/bevy_render/src/camera/camera.rs

```diff
@@ -370,7 +370,7 @@ impl CameraRenderGraph {
 
 /// The "target" that a [`Camera`] will render to. For example, this could be a [`Window`](bevy_window::Window)
 /// swapchain or an [`Image`].
-#[derive(Debug, Clone, Reflect)]
+#[derive(Debug, Clone, Reflect, PartialEq, Eq, Hash, PartialOrd, Ord)]
```

Comment:
I'm not sure if Ord can be implemented here properly. Are we potentially arbitrarily stating that all `Window` variants are less than `Image` variants? Does that even make sense?

Reply:
Upon checking the docs for the stdlib, it occurred to me the exact ordering behavior of the derive macros are not well documented/defined. Filed https://github.com/rust-lang/rust/issues/109946 as a follow-up.

If enum variants matter, and we want a first-to-last ordering for render targets, I'm tempted to say we should put `Image` variants before `Window`, since that tends to be how you'd expect it to be sorted in a rendering order, and we should probably document that too. This can be done in this PR or as a follow-up.